### PR TITLE
updated docs to v1.10

### PR DESCRIPTION
**Pull request by ozgunozerk**
Fixes #165

### CODE REVIEW

File: docs/modules/ROOT/pages/pallets/assets.adoc [hunk 1]
```diff
@@ -17,7 +19,7 @@ The `pallet_assets` module is designed to manage and manipulate custom fungible
 ** `Admin` - The account that has administrative privileges over an asset, such as: `minting`, `burning`, `freezing`, and `thawing`.
 ** `Sufficiency` - The amount of an asset that is required to be held in an account in order to perform certain operations, such as: `transfers`, `freezing`, and `thawing`.
 
-== Config link:https://github.com/paritytech/polkadot-sdk/blob/release-polkadot-v1.7.0/substrate/frame/assets/src/lib.rs#L231[{github-icon},role=heading-link]
+== Config
```
[the remark this answers is not in the document]
Comment (KitHat) [reply docs/modules/ROOT/pages/pallets/assets.adoc:22]: Both links were merged into one on line 9

**Review by KitHat — APPROVED** [review]

File: docs/modules/ROOT/pages/pallets/assets.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (ozgunozerk) [reply docs/modules/ROOT/pages/pallets/assets.adoc:22]: I can revert, but here is my reasoning: with each update, we have to go through all the links and update the lines. Not that helpful to other devs (they for sure can use `ctrl+f`), costs a lot of our time, and is very error prone.

So, I decided to give a single link (without lines). For me, it was a sweet spot.

**Review by KitHat — CHANGES_REQUESTED** [review]
I know, that it is out of scope for this PR, but @ozgunozerk could you please update the `docs/antora.yml` to be `Substrate Runtimes` instead of `Substrate Runtime`?
It will help us a lot during the integration to main docs site.

**Comment by ozgunozerk** [comment]
Sure, but im afk at the moment. Shall we merge this and tomorrow morning i can create a PR for that? And it will be more granular? Or you can do it, however you please :)

**Comment by KitHat** [comment]
It waits till tomorrow :)

**Review by KitHat — APPROVED** [review]
LGTM!

**Pull request merged**